classes/class-wc-rest-connect-account-settings-controller.php

```diff
@@ -58,4 +67,10 @@ public function post( $request ) {
 		return new WP_REST_Response( array( 'success' => true ), 200 );
 	}
 
+	private function can_user_manage_payment_methods() {
+		global $current_user;
+		$master_user = WC_Connect_Jetpack::get_master_user();
+		return WC_Connect_Jetpack::is_development_mode() ||
+			( is_a( $master_user, 'WP_User' ) && $current_user->ID == $master_user->ID );
```

Comment:
@nabsul @marcinbot I'm not sure that the changes discussed here ever made it through in the merged PR? Maybe we should open a new PR to update?